public/docs/_examples/template-syntax/dart/lib/app-component.html

```diff
@@ -0,0 +1,654 @@
+<!-- #docregion my-first-app -->
+<h3>My First Angular Application</h3>
+<!-- #enddocregion my-first-app -->
+
+<!-- Interpolation and expressions -->
+<hr><h2>Interpolation</h2>
+
+<!-- #docregion first-interpolation -->
+<p>My current hero is {{currentHero.firstName}}</p>
+<!-- #enddocregion first-interpolation -->
+
+<!-- #docregion title+image -->
+<h3>
+    {{title}}
+    <img src="{{heroImageUrl}}" style="height:30px">
+</h3>
+<!-- #enddocregion title+image -->
+
+<!-- #docregion sum-1 -->
+<!-- "The sum of 1 + 1 is 2" -->
+<p>The sum of 1 + 1 is {{1 + 1}}</p>
+<!-- #enddocregion sum-1 -->
+
+<!-- #docregion sum-2 -->
+<!-- "The sum of 1 + 1 is not 4" -->
+<p>The sum of 1 + 1 is not {{1+1+getVal}}</p>
+<!-- #enddocregion sum-2 -->
+
+
+<!-- New Mental Model -->
+<hr><h2>New Mental Model</h2>
+
+<!-- #docregion img+button -->
+<div class="special">Mental Model</div>
+<div><b>{{currentHero.fullName}}</b></div>
+<img src="assets/images/hero.png">
+<button disabled>Save</button>
+<!-- #enddocregion img+button -->
+<br><br>
+
+<div>
+    <!-- #docregion hero-detail-1 -->
+    <div class="special">Mental Model</div>
+    <hero-detail></hero-detail>
+    <!-- #enddocregion hero-detail-1 -->
+</div>
+<br><br>
+
+<div>
+    <!-- #docregion disabled-button-1 -->
+    <!-- Bind button disabled state to `isUnchanged` property -->
+    <button [disabled]="isUnchanged">Save</button>
+    <!-- #enddocregion disabled-button-1 -->
+</div>
+<br><br>
+
+<div>
+    <!-- #docregion property-binding-syntax-1 -->
+    <img [src] = "heroImageUrl">
+    <hero-detail [hero]="currentHero"></hero-detail>
+    <div [ngClass] = "{selected: isSelected}"></div>
+    <!-- #enddocregion property-binding-syntax-1 -->
+</div>
+<br><br>
+
+<!-- #docregion event-binding-syntax-1 -->
+<button (click) = "onSave()">Save</button>
+<hero-detail (deleted)="onHeroDeleted()"></hero-detail>
+<div myClick (myClick)="clicked=$event">click me</div>
+<!-- #enddocregion event-binding-syntax-1 -->
+<!--{{clicked}}-->
+<br><br>
+
+<div>
+    <!-- #docregion 2-way-binding-syntax-1 -->
+    <input [(ngModel)]="heroName">
+    <!-- #enddocregion 2-way-binding-syntax-1 -->
+    Hero Name: {{heroName}}
+</div>
+<br><br>
+
+<!-- #docregion attribute-binding-syntax-1 -->
+<button [attr.ariaLabel]="help">help</button>
+<!-- #enddocregion attribute-binding-syntax-1 -->
+<br><br>
+
+<!-- #docregion class-binding-syntax-1 -->
+<div [class.special]="isSpecial">Special</div>
+<!-- #enddocregion class-binding-syntax-1 -->
+<br><br>
+
+<!-- #docregion style-binding-syntax-1 -->
+<button [style.color] = "isSpecial ? 'red' : 'green'">
+    <!-- #enddocregion style-binding-syntax-1 -->
+    button</button>
+
+<hr><h2>kebab-case</h2>
+<!-- #docregion bad-mixed-case -->
+<!-- BAD: mixed case target names don't work
```

Comment:
What does this mean? I got the impression this is the preferred style currently?